<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1079</width>
    <height>789</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1041</width>
      <height>731</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Device Setting</string>
     </attribute>
     <widget class="QComboBox" name="comboSpiSpeed">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>63</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboSpiMode">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>93</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineReadTimeout">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>133</y>
        <width>61</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineWriteTimeout">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>163</y>
        <width>61</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnApplyConfig">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>203</y>
        <width>121</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Set</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnConnect">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>121</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>SPI RD/WR</string>
     </attribute>
     <widget class="QLineEdit" name="lineReadCmd">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>101</y>
        <width>141</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Courier New</family>
       </font>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineReadBytes">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>381</y>
        <width>41</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineWriteCmd">
      <property name="geometry">
       <rect>
        <x>539</x>
        <y>99</y>
        <width>151</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Courier New</family>
       </font>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSpiRead2">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>425</y>
        <width>101</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>SPI Read Set</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSpiWrite">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>232</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>SPI Write</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="textSpiReadResult">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>450</y>
        <width>1001</width>
        <height>211</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Courier New</family>
       </font>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineSpiDelayMs">
      <property name="geometry">
       <rect>
        <x>956</x>
        <y>41</y>
        <width>51</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>874</x>
        <y>21</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Delay Time (ms) after CMD</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>103</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Read CMD(Hex)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>386</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Read Length</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>102</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Write CMD(Hex)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>142</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Write Data(Hex)</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnClearResult">
      <property name="geometry">
       <rect>
        <x>920</x>
        <y>670</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear Result</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineDummyCount">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>73</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>74</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Wake-up Number</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineDummyCount_2">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>75</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>76</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Wake-up Number</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="chkReadRepeatEnable">
      <property name="geometry">
       <rect>
        <x>23</x>
        <y>428</y>
        <width>81</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Read Repeat</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineReadRepeatCount">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>428</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineReadRepeatInterval">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>428</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="chkWriteRepeatEnable">
      <property name="geometry">
       <rect>
        <x>465</x>
        <y>282</y>
        <width>91</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Write Repeat</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineWriteRepeatCount">
      <property name="geometry">
       <rect>
        <x>555</x>
        <y>282</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineWriteRepeatInterval">
      <property name="geometry">
       <rect>
        <x>621</x>
        <y>282</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>113</x>
        <y>414</y>
        <width>47</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>Count.</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>557</x>
        <y>268</y>
        <width>47</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>Count.</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>411</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Interval(ms)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>626</x>
        <y>265</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Interval(ms)</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="textWriteData">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>139</y>
        <width>481</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Courier New</family>
       </font>
      </property>
     </widget>
     <widget class="QComboBox" name="comboReadCmdList">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>141</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnLoadReadCmdList">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>142</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Load CMD List</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboWriteCmdList">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>99</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnLoadWriteCmdList">
      <property name="geometry">
       <rect>
        <x>701</x>
        <y>76</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Load CMD List</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboWriteDataList">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>232</y>
        <width>221</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnLoadWriteDataList">
      <property name="geometry">
       <rect>
        <x>856</x>
        <y>232</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Load Data List</string>
      </property>
     </widget>
     <widget class="QListView" name="listViewReadCmds">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>141</y>
        <width>181</width>
        <height>281</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Courier New</family>
       </font>
      </property>
     </widget>
     <widget class="QComboBox" name="comboReadCmdSet">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>101</y>
        <width>181</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnLoadReadCmdSet">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>71</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Load CMD Set List</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSpiRead">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>171</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>SPI Read One</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>5</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="title">
       <string>Direction</string>
      </property>
      <widget class="QRadioButton" name="rdoNorth">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>85</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>North</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="rdoSouth">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>30</y>
         <width>85</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>South</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1079</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
